tools/screenshot-test/src/app/firebase.service.ts

```diff
@@ -0,0 +1,152 @@
+import {Injectable} from '@angular/core';
+import * as firebase from 'firebase';
+
+const request = require('request');
+const config = require('../config.json');
+
+import {ScreenshotResult} from './screenshot-result';
+
+/** The service to fetch data from firebase database */
+@Injectable()
+export class FirebaseService {
+
+  /** The current user */
+  user: any;
+
+  /** The screenshot results */
+  screenshotResult: ScreenshotResult;
```

Comment:
Since this captures the result for _all_ screenshots in the PR, how about calling this class `ScreenshotResultsSummary` (and the member `screenshotResultsSummary`)?